samples/bluetooth/mesh/ble_peripheral_lbs_coex/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -7,3 +7,7 @@
 # Application overlay - nrf54l15
 
 CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE=n
+CONFIG_NVS=n
+CONFIG_NVS_LOOKUP_CACHE=n
+CONFIG_SETTINGS_NVS_NAME_CACHE=n
+CONFIG_ZMS=y
```

Comment:
Maybe we should add these too for all the samples:
CONFIG_SETTINGS_ZMS_SECTOR_COUNT=8
CONFIG_ZMS_LOOKUP_CACHE=y
CONFIG_ZMS_LOOKUP_CACHE_SIZE=512
CONFIG_SETTINGS_ZMS_NAME_CACHE=y
CONFIG_SETTINGS_ZMS_NAME_CACHE_SIZE=512
CONFIG_ZMS_LOOKUP_CACHE_FOR_SETTINGS=y

Reply:
Where did you get these values? Are these based of the testing that Omkar did perhaps?